Makefile

```diff
@@ -15,8 +15,12 @@ ifdef CI
 	RUNNER_VERSION := $(RUNNER_SEMVER)
 	PROBE_VERSION := $(PROBE_SEMVER)
 else
-	RUNNER_VERSION := "$(RUNNER_SEMVER)-$(shell date +%s)"
-	PROBE_VERSION := "$(PROBE_SEMVER)-$(shell date +%s)"
+	COMMIT_SHA := $(shell git rev-parse HEAD)
+	WORKING_TREE_SHA := $(shell git ls-files -m -o --exclude-standard \
+		| while read -r file; do stat -c '%n %a' $${file}; done \
+		| sha1sum | tr -s ' ' | tr -d ' -')
```

Comment:
I just realized the call to `stat -c …` fails in macOS, because the flag is not `-c` but rather `-f`. I'll try to fix it in another PR.

Also, the following should do the same with fewer calls:

```ash
git ls-files -m -o --exclude-standard | xargs stat -c '%n %a' | sha1sum | tr -d' ' -f1
```